Tests/AbstractTests/BoundedSemilatticeTests.generated.swift

```diff
@@ -0,0 +1,48 @@
+// Generated using Sourcery 0.7.2 — https://github.com/krzysztofzablocki/Sourcery
```

Comment:
It would be cool to not commit the generated code with Sourcery -- we could just make it a requirement to build the project. The problem with committing artifacts is that it makes PRs and reading code really messy (because people have to sift through generated code). What do you think?